src/mixins/org-person-mixin.ts

```diff
@@ -0,0 +1,94 @@
+import { Component, Vue } from 'vue-property-decorator'
+import { OrgPersonIF } from '@/interfaces'
+import { ActionTypes, PartyTypes, RoleTypes } from '@/enums/'
+
+/**
+ * Mixin that provides common OrgPerson methods.
+ */
+@Component({})
+export default class OrgPersonMixin extends Vue {
```

Comment:
Some of these methods were used in multiple components, so I extracted them here.